docs/_scripts/quickstart.sh

```diff
@@ -0,0 +1,19 @@
+# This script will execute the component scripts and ensure that the documented examples
+# work as expected.
```

Comment:
Idea for another day:

It might be worth running quickstart.sh when building the docs, to test it out. As a possible side benefit, we will populate the db, which might allow us to include examples in the REST API docs.